examples/sdk/auth-expiration-typescript-example/assets/styles/reset.css

```
@@ -0,0 +1,129 @@
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
```

Stylelint (17.6.0)

[error] Unexpected shorthand "font" after "font-size" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)